### PR TITLE
Add first keynote

### PR DESCRIPTION
**Pull request by cmaureir**

### CODE REVIEW

**Comment by cmaureir** [comment]
This proposal, extends  https://github.com/pyladies/global-conference/pull/62 so we can have the keynotes on top of the speaker lists, in a more prominent way.

The bio was reduced to the first paragraph.

Screenshot 2023-10-30 at 22-43-58 PyLadiesCon 2023

File: generate_site/keynotes.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+Marlene Mhangami,Marlene@fosstodon.org,marlene_zw,marlene_zw,marlenemhangami,"img/keynote/marlene.jpg","Marlene is a Zimbabwean software engineer, explorer, and speaker based in the city of Harare. She is a previous director and vice-chair for the Python Software Foundation and is currently serving as the vice-chair of the Association for Computing Machinery practitioner board. In 2017, she co-founded ZimboPy, a non-profit organization that gives Zimbabwean young women access to resources in the field of technology. She is also the previous chair of PyCon Africa and is an advocate for women in tech on the continent. Professionally, Marlene is currently working as a Developer Advocate at Voltron Data."
```
Comment (Mariatta) [inline generate_site/keynotes.csv:4]:
```suggestion
Marlene Mhangami,https://fosstodon.org/@marlene,marlene_zw,marlene_zw,marlenemhangami,"img/keynote/marlene.jpg","Marlene is a Zimbabwean software engineer, explorer, and speaker based in the city of Harare. She is a previous director and vice-chair for the Python Software Foundation and is currently serving as the vice-chair of the Association for Computing Machinery practitioner board. In 2017, she co-founded ZimboPy, a non-profit organization that gives Zimbabwean young women access to resources in the field of technology. She is also the previous chair of PyCon Africa and is an advocate for women in tech on the continent. Professionally, Marlene is currently working as a Developer Advocate at Voltron Data."
```

File: generate_site/keynotes.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+Jimena Bermúdez,Jimena_y_yo@mastodon.social,Jimena_y_yo,,jimena-eb,"img/keynote/jimena.jpeg","Jimena is the esteemed president of Python Spain and a computer engineer who takes pride and joy in her profession. Although she was once drawn to the realms of design, UI, and UX, her true passion lies in coding and architectural design. A gifted communicator, Jimena has served as a web development instructor for individuals transitioning careers. She is an avid mentor, community speaker, and event organizer, holding active memberships in both Python Spain and PyLadies. With numerous publications emphasizing the importance of coding education and industry diversity, Jimena can be best described as a 'Full Stack developer ardently championing diversity, equity, and inclusion (DEI) from a tech standpoint and cherishing her role in the community.'"
```
Comment (Mariatta) [inline generate_site/keynotes.csv:3]:
```suggestion
Jimena Bermúdez,https://mastodon.social/Jimena_y_yo,Jimena_y_yo,,jimena-eb,"img/keynote/jimena.jpeg","Jimena is the esteemed president of Python Spain and a computer engineer who takes pride and joy in her profession. Although she was once drawn to the realms of design, UI, and UX, her true passion lies in coding and architectural design. A gifted communicator, Jimena has served as a web development instructor for individuals transitioning careers. She is an avid mentor, community speaker, and event organizer, holding active memberships in both Python Spain and PyLadies. With numerous publications emphasizing the importance of coding education and industry diversity, Jimena can be best described as a 'Full Stack developer ardently championing diversity, equity, and inclusion (DEI) from a tech standpoint and cherishing her role in the community.'"
```

**Review by Mariatta — APPROVED** [review]
Thank you!

File: generate_site/templates/index-base.html [hunk 3]
```diff
@@ -69,6 +69,46 @@ <h3 class="display-6 special-font pink">About the conference</h2>
...
+            <div class="col-12 col-xl-6 d-flex">
+              <div class="card card-keynote my-3 bg-opacity-50">
+                <img src="{{ keynote['photo'] }}" class="card-img-top w-100" alt="{{ keynote['name'] }} Photo">
+                <div class="card-img-overlay h-100 d-flex flex-column justify-content-end text-start">
+                  <h5 class="card-title text-white">{{ keynote['name'] }}</h5>
+                </div>
+                <div class="card-body w-100">
+                  <p class="card-text text-start">{{ '.'.join(keynote['bio'].split('.')[:2]) }}.</p>
+                  <div class="social">
+                    {% if keynote['mastodon'] %}
```
Comment (cmaureir) [inline generate_site/templates/index-base.html:88]: @Mariatta here is some logic in the template to transform the `username@domain` from mastodon, to `https://{domain}/@{username}`

Comment (Mariatta) [reply generate_site/templates/index-base.html:88]: Yup it seems to work now. I commented before you added your fix. My suggestions can be ignored. Thanks.

File: generate_site/keynotes.csv (hunk 2 above)
Comment (Mariatta) [reply generate_site/keynotes.csv:3]: Doesn't apply anymore.

**Comment by Mariatta** [comment]
Keynote names are floating over the bio on narrow width screen.

Screenshot 2023-10-31 at 10 18 36 AM

**Comment by Mariatta** [comment]
I tested with various devices. Looks great.

**Review by Mariatta — APPROVED** [review]

**Pull request merged**